utils/mariadb_charset/charset.cpp

```diff
@@ -0,0 +1,22 @@
+#include "mariadb_charset/collation.h"
```

Comment:
There is no license header. Check other files plz.

Reply:
++